MAINTAINERS.md

```diff
@@ -86,9 +86,10 @@ To update the lorri version in [nixpkgs][]:
 
    > @GrahamcOfBorg build lorri.tests
 
-2. **`nixos-19.09`**: _after_ the first PR has been merged into `master`,
+2. **`nixos` stable**: _after_ the first PR has been merged into `master`,
    follow the [backporting procedure][nixpkgs-backporting] to create a PR
-   against `release-19.09`; see for example [NixOS#77432][nixos-stable-pr].
+   against `release-<stable-release-date>`;
+   see for example [NixOS#77432][nixos-stable-pr].
```

Comment:
If we support multiple stable release channels, then these instructions need to be clearer IMO. Would the backporting need to happen in a particular order?

Reply:
I would only ever backport to the latest stable and skip on major versions.